### ISSUE TITLE
API error, NoRecordInDatabase

### ISSUE THREAD

**Issue by dotcomfy** [gboudreau/nissan-connect-php#34]
This isn't really an issue with nissan-connect-php as such, but since I lack a better place to discuss it, I'm bringing it up here.

I can update the battery status, from PHP, from the NissanConnect web site, and even with the official Android app.

However, I cannot start climate control. It sends the command via https://gdcportalgw.its-mo.com/api_v190426_NE/gdc/ACRemoteRequest.php, but then gets an error message when looking for the result via ACRemoteResult.php.

`Response received: {"status":"-6004","ErrorCode":"-6004","ErrorMessage":"NoRecordInDatabase"}
`
Does anyone else have the same problem? I've tried with Nissan customer service, but they have no idea, and I think the customer service people don't even have a way of passing the issue on to someone who has enough access/knowledge to be able to troubleshoot.

**Comment by dotcomfy** [comment]
@gboudreau if you don't want this issue to "litter" your issues list, then feel free to get rid of it.

**Comment by AlessandroEmm** [comment]
@dotcomfy were you ever able to solve this?

**Comment by dotcomfy** [comment]
@AlessandroEmm Yes, this was eventually resolved by deleting my account, setting up a new one, and I think I had to get Nissan to associate it with the car. I had it working until some time in July, when we sold the car to moved overseas.
